devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/AnalyticsExtension.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.gradle.extension;
+
+import java.util.Map;
+
+public class AnalyticsExtension {
```

Comment:
Not sure whether another extension is a good way here. IIUC `GraalVMInfo` shall be taken from the Quarkus native build's `AugmentResult`, which is available in `io.quarkus.gradle.tasks.worker.BuildWorker`
Guess it's better to send the analytics from `BuildWorker`, making this class and probably all added tasks superfluous.